src/client/components/logout/logout.component.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import Popup from 'reactjs-popup';
+import './logout.style.css';
+import PropTypes from 'prop-types';
+
+const userName = 'Servet';
+
+export default function Logout() {
+  const popupView = (closeAction) => (
+    <PopupModal title="LOG OUT" closeAction={closeAction}>
+      <LogoutContent>
+        <LogoutActions closeAction={closeAction} />
+      </LogoutContent>
+    </PopupModal>
+  );
+
+  return (
+    <Popup trigger={PopupTrigger} modal>
+      {popupView}
+    </Popup>
+  );
+}
+
+const PopupTrigger = () => (
+  <button type="submit" className="logout-button">
+    LOG OUT
+  </button>
+);
+
+const PopupModal = ({ closeAction, children, title }) => {
+  return (
+    <div className="modal">
+      <button type="submit" className="close" onClick={closeAction}>
+        &times;
+      </button>
+      <div className="header">{title}</div>
+      {children}
+    </div>
+  );
+};
+
+const LogoutContent = ({ children }) => {
+  return (
+    <>
+      <div className="content">
+        <p>
+          Hi {userName} <br />
+          Do you really want to Logout ?{' '}
+        </p>
+      </div>
+      {children}
+    </>
+  );
+};
+
+const LogoutActions = ({ closeAction }) => (
+  <div className="actions">
+    <button className="confirm-logout" type="submit" onClick="{() => {}}">
```

Comment:
Shouldn't you pass a "logout function" here as a prop?

Reply:
It's unknown by now

Reply:
Alright, it is fine to keep like this but would rather have it as a prop and pass an "empty function" here. @senner008 what is your opinion on this? Then we shouldn't have to change the file again 🤞

Reply:
you decide...